modules/importer/import-helpers.js

```diff
@@ -2447,14 +2449,16 @@ export default class ImportHelpers {
             if (compendiumEntry) {
               if (compendiumEntry?.type === "itemmodifier") {
                 const descriptor = duplicate(compendiumEntry);
+                console.log("this is a log")
+                  console.log(descriptor)
```

Comment:
```suggestion
```
I totally should just use the github suggestion thing so you can auto-commit it, without having to go dig through that file. I'll remember one of these days.